website/routes.py

```diff
@@ -1412,6 +1412,16 @@ def make_url_map(app):
         ),
 
         Rule(
+            [
+                '/project/<pid>/file_subscriptions/',
+                '/project/<pid>/node/<nid>/file_subscriptions/',
+            ],
+            'get',
```

Comment:
Thanks! I have removed them for the time being. If they are going to be used again they will be using api v2 and the hierarchy suggested.